docs/guide/piece-pages.md

```diff
@@ -1,257 +1,208 @@
+
+This method allows you to name the module whatever you want since you are specifically identifying a piece type. Set the [`pieceModuleName` option](/reference/module-api/module-options.md#piecemodulename) to the piece type name and Apostrophe make the right connection.
+
+```javascript
+// modules/blog-page/index.js
 module.exports = {
   extend: '@apostrophecms/piece-page-type',
   options: {
-    label: 'Product Index Page'
+    label: 'Blog page',
+    pieceModuleName: 'article'
   }
 };
 ```
 
+Either method works well and you may find both options useful depending on the situation.
+
+### Add template files and instantiate
+
+Piece page types use two templates, both added in the module's `views` directory (e.g., `modules/article-page/views/`).
+
+| Template file name | What is it? |
+| ------------------ | ----------- |
+| `index.html` | Template for listing pieces (the **"index page**) |
+| `show.html` | Template to display an individual piece (a **"show page"**) |
+
+We'll review each template's features next.
+
+Once those template files exist, you would **add this to the `app.js` configuration** [like any other module](/guide/modules.html#setting-up-a-module).
+
+🛑 Hold up. ✋
+
+You've reviewed the [page type guide](/guide/pages.md), right? The sections below will highlight the special features of index and show page templates. For general page template syntax, see that page type guide.
+
+## The index page template
+
+Index page templates look very similar to other page templates.
+
 ```django
-{# modules/product-page/views/index.html #}
+{# modules/article-page/views/index.html #}
 
-{% import '@apostrophecms/pager:macros.html' as pager with context %}
 {% extends "layout.html" %}
+{% import '@apostrophecms/pager:macros.html' as pager with context %}
 
 {% block main %}
-  {% for product in data.pieces %}
-    <h2>
-      <a href="{{ product._url }}">{{ product.title }}: {{ product.price }}</a>
-    </h2>
-    <section>{% area product, 'description' %}</section>
+  <h1>{{ data.page.title }}</h1>
+
+  {% for article in data.pieces %}
+    <article>
+      <h2>
+        <a href="{{ article._url }}">{{ article.title }}</a>
+      </h2>
+    </article>
   {% endfor %}
 
-  {# The pager macro now takes a `class` option to set `class` attributes. #}
   {{ pager.render({
     page: data.currentPage,
     total: data.totalPages,
-    class: 'my-pager-class'
+    class: 'blog-pager'
   }, data.url) }}
 {% endblock %}
 ```
 
-```django
-{# modules/product-page/views/show.html #}
-{% extends "layout.html" %}
-{% set product = data.piece %}
+### `data.pieces` and other unique `data` properties
 
-{% block main %}
-  {# The layout already output the title for us #}
-  <h4>Price: {{ product.price }}</h4>
-  <section>{% area product, 'description' %}</section>
-{% endblock %}
+The first new thing here is the `import` statement, but we'll get back to that. Let's talk about the **loop over `data.pieces`**.
```

Comment:
Do we go over this or did I miss it. I'm always wary of saying 'hey check out this, but ignore it for now'